google_shopping.js

```diff
@@ -1,5 +1,87 @@
 var data = require("./products.json")
 
-// Write your solutions below
+ 
+
+//Question 1:
+console.log(data["items"].length);
+
+//The same count information is seen labeled as "current item count at the top"
+console.log(data["currentItemCount"]); 
+
+//Question 2:
+
+for (var availability in inventories) {
+	if (inventories[availability]) === "backorder") {
+		console.log(product)
+	}
+	else {
+		console.log(null);
+	}
+}
+
+//items[15].product.inventories[0].availability;
+
+
+
+for (var i = 0; i < data["items"].length; i++ ) {
+	if (data["items"][i].product.inventories[0]["availability"] === "backorder") {
+		console.log(data["items"][i].product.title);
+		// console.log(i)
+	}
+}
+
```

Comment:
This is good. You could also make a counter variable and increment it at each match:

```var numberOfBackorderProducts = null;```
```for (var i = 0; i < data["items"].length; i += 1) {```
	```if (data["items"][i].product.inventories[0].availability == 'backorder') {```
	```numberOfBackorderProducts++```
	```};	```
```};```